lib/publiccloud/instance.pm

```diff
@@ -438,9 +441,11 @@ sub softreboot {
     }
     my $shutdown_time = time() - $start_time;
     die("Waiting for system down failed!") unless ($shutdown_time < $args{timeout});
-    my $bootup_time = $self->wait_for_ssh(timeout => $args{timeout} - $shutdown_time,
+    my $bootup_time = $self->wait_for_ssh(
+        timeout => $args{timeout} - $shutdown_time,
         username => $args{username},
         ignore_wrong_pubkey => $args{ignore_wrong_pubkey});
+    record_info('UPTIME', $self->ssh_assert_script_run(cmd => 'uptime'));
```

Comment:
PR 16890 will add to the test results more information about system status and elapsed time, in test details `WAIT CHECK` boxes.